models/helper/LogHelper.py

```diff
@@ -0,0 +1,77 @@
+import logging
+
+class Logger:
+    logger = None
+
+    def __init__(self):
+        pass
+
+    @classmethod
```

Comment:
@emreeroglu, not used this before. What does it do?

Reply:
This binds method with class itself not the instance of the classes.
We are not creating any instances of Logger class, we are just using it everywhere.